contracts/package.json

```diff
@@ -23,6 +23,7 @@
     "abi/"
   ],
   "devDependencies": {
+    "@defi-wonderland/smock": "^2.0.7",
```

Comment:
We're already using [Waffle's mock contract](https://ethereum-waffle.readthedocs.io/en/latest/mock-contract.html) in [some places](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/package.json#L39). (We plan to pull that repo into our main repo, just haven't gotten around to it.) We should try to keep a consistent stack, and use the same mocking libraries. I believe the APIs are pretty similar, so it shouldn't be too much work to switch over.

Reply:
I removed smock and imported `@ethereum-waffle/mock-contract` for mocking the L1 gas feed in the ArbitrumValidator test. Unfortunately it didn't work for mocking the Inbox contract because waffle mock function matchers don't work with hardhat: https://github.com/nomiclabs/hardhat/issues/1135. To resolve this, I created a `MockArbitrumInbox.sol` contract just for testing.